src/components/common/navbar/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect } from "react";
+import { Icon } from "@components/common";
+import {
+  openMenu,
+  closeMenu,
+  setActiveLink,
+} from "@/state/features/menu/menuSlice";
+
+import { Link, useLocation } from "react-router-dom";
+import { useAppDispatch, useAppSelector } from "@/state/app/hooks";
+
+import styles from "./Navbar.module.css";
+
+const Navbar = () => {
+  const dispatch = useAppDispatch();
+  const { menuOpen, activeLink } = useAppSelector((state) => state.menu);
+
+  const location = useLocation();
+
+  const handleMenu = (status: boolean) => {
+    status ? dispatch(openMenu()) : dispatch(closeMenu());
+  };
+
+  const handleActiveLink = (activeLink: string) => {
+    dispatch(setActiveLink(activeLink));
+    dispatch(closeMenu());
+  };
+
+  const navLinks = [
+    { href: "/about", name: "About" },
+    //I'm not including "Contact" here as it's a Modal and it's not supposed to change url path
+    //However, array structure is made in case more links are needed in the future
+  ];
+
+  useEffect(() => {
+    location.pathname === "/about" && activeLink !== "/about"
```

Comment:
WDYT if you replace those routes with the constants that I mentioned above?

Reply:
Finally I end up just using `location.pathname` to adjust styles instead of using `activeLink` state. 
But I will I like the idea of using constants or enums for the paths. I will add them thanks!